misc-scripts/xref_mapping/XrefMapper/XrefLoader.pm

```diff
@@ -65,21 +65,18 @@ sub update{
   #####################################
   print "Deleting all PROJECTIONs from this database\n" if $verbose;
 
-  my $sql = "DELETE es FROM xref x, external_synonym es WHERE x.xref_id = es.xref_id and x.info_type = 'PROJECTION'";
+  my $sql = "TRUNCATE table ontology_xref";
```

Comment:
Truncate would reset the auto_increment counter - hope that value is not being used anywhere else

Reply:
not to my knowledge